css/index.css

```diff
@@ -1 +1,258 @@
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/* MIXINS */
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+  /*  background-color:red; */
+}
+html,
+body {
+  height: 100%;
+  background-color: white;
+}
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+.container h1 {
+  margin-top: 2rem;
+  text-align: center;
+  font-size: 3rem;
+}
+.nav-bar {
+  display: flex;
+  margin-top: 2rem;
+  background: grey;
+  color: white;
+  /* NAV BAR MEDIA QUERY */
+}
+.nav-bar a {
+  margin: auto;
+}
+.nav-bar a:hover {
+  color: black;
+}
+@media (max-width: 500px) {
+  .nav-bar {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    background: grey;
+  }
+  .nav-bar a {
+    padding-bottom: 1rem;
+    padding-top: 1rem;
+    border: 2px solid black;
+    width: 100%;
+    text-align: center;
+  }
+}
+.intro {
+  display: flex;
+  border: 2px solid black;
+  margin-top: 2rem;
+  background: lightgrey;
+}
+.intro img {
+  border: 2px solid black;
+  margin: 20px;
+}
+.intro p {
+  margin-top: 3rem;
+}
+.qualifications {
+  border: 2px solid black;
+  margin-top: 2rem;
+  display: flex;
+  flex-display: row;
+  align-items: center;
+  justify-content: center;
+  padding: 0.4rem;
+  background: lightgrey;
+  /* QUALIFICATIONS MEDIA QUERY */
+}
+.qualifications h2 {
+  font-size: 1.5rem;
+  margin-bottom: 1rem;
+  margin-top: 1rem;
+  border: 2px solid black;
+  padding: 0.3rem;
+  background: grey;
+}
+.qualifications br {
+  margin-bottom: 0.6rem;
+}
+.qualifications .skills {
+  margin-right: 0.6rem;
+  max-width: 50%;
+}
+.qualifications .history {
+  margin-left: 0.6rem;
+  max-width: 50%;
+}
+@media (max-width: 500px) {
+  .qualifications {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    justify-content: center;
+  }
+  .qualifications p {
```

Comment:
So for here instead of doing `.qualifications p` use the power of less to just nest it inside of `.qualifications` in fact everything that is nested in qualifications in the html can be nested in it in LESS.

For example, the qualifications class would be changed like

```
.qualifications {
  border: 2px solid black;
  margin-top: 2rem;
  display: flex;
  flex-display: row;
  align-items: center;
  justify-content: center;
  padding: 0.4rem;
  background: lightgrey;
     h2 {
  font-size: 1.5rem;
  margin-bottom: 1rem;
  margin-top: 1rem;
  border: 2px solid black;
  padding: 0.3rem;
  background: grey;
  }
   br {
    margin-bottom: 0.6rem;
  }
  @media (max-width: 500px) {
    flex-wrap: wrap;
    flex-direction: column;
  }
}
```